tests/util_test.py

```diff
@@ -117,5 +118,19 @@ def test_lines_modified_by_commit(self):
              frozenset())
         )
 
+
+    def test_last_commit_to_line(self):
+        repo = get_repo('https://github.com/squaresLab/blameandshame-test-repo')
+        self.assertEqual(last_commit_to_line(repo, 'file-one.txt', 1,
+                                             repo.commit('9ca70f7')),
+                         repo.commit('922e13d'))
+        self.assertEqual(last_commit_to_line(repo, 'file-one.txt', 1,
+                                             repo.commit('422cab3')),
+                         None)
+        self.assertEqual(last_commit_to_line(repo, 'file-one.txt', 1,
```

Comment:
Any chance we could test it at a different line?

Reply:
I'll get that done.